src/main/java/com/google/sps/tool/ResponseHandler.java

```diff
@@ -0,0 +1,17 @@
+package com.google.sps.tool;
+
+import java.io.IOException;
+import javax.servlet.http.HttpServletResponse;
+import com.google.gson.Gson;
+import java.util.Map;
+import java.util.HashMap;
+
+public class ResponseHandler {
+  
+  public static void sendErrorMessage(HttpServletResponse response, String error) throws IOException {
+    Map<String, String> jsonErrorInfo = new HashMap<>();
+    jsonErrorInfo.put("error", error);
```

Comment:
Above we used `ResponseHandler.sendErrorMessage`; here we send a JSON error. Why are we returning errors to the client inconsistently?

Reply:
I was hoping to send the error formatted within a JSON object. That way, on the front end, we can parse the error and signal like popup or having a red animation in the edit folder screen, rather than redirecting them to a configured error page. Should I change the naming of my method call?

Reply:
Oh I see; I thought ResponseHandler was some other class. This is fine